postcss.config.cjs

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  syntax: 'postcss-scss',
+  plugins: [
+    require('postcss-modules')({
+      generateScopedName: '[name]__[local]___[hash:base64:5]',
```

Comment:
why do we need to customize the scoped name here? could we somehow reuse an existing algorithm postcss already has for naming things?